pkgs/applications/editors/neovim/neovide/Cargo.lock.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Cargo.lock b/Cargo.lock
+index 6607b0e..2e05fd0 100644
+--- a/Cargo.lock
++++ b/Cargo.lock
+@@ -1339,7 +1339,7 @@ checksum = "e1bcdd74c20ad5d95aacd60ef9ba40fdf77f767051040541df557b7a9b2a2121"
+ 
+ [[package]]
+ name = "neovide"
+-version = "0.10.1"
++version = "0.10.2"
```

Comment:
wouldn't it be easier to use a regex rather than a patch ? i am ok with both. Did they just forget to tag the repo ?

Reply:
I could remove this patch, because this issue was fixed upstream with a new release 0.10.3